docs/supported_alpha.md

```diff
@@ -0,0 +1,1141 @@
+﻿---
```

Comment:
There is a BOM (byte order mark), which seems to prevent parsing metadata. Removing it seems to fix the issue:
```diff
diff --git a/docs/supported_alpha.md b/docs/supported_alpha.md
index 0b75caf..8585a93 100644
--- a/docs/supported_alpha.md
+++ b/docs/supported_alpha.md
@@ -1,4 +1,4 @@
-<EF><BB><BF>---
+---
 id: supported_alpha
 title: Functions Sorted Alphabetically
 sidebar_label: Functions - Alpha Sort
```